packages/js/data/changelog/update-add-is_installed-and-plugins_page_skipped

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Add is_installed and plugins_page_skipped type
```

Comment:
We might also need to include the "installPlugins" change log here?

Reply:
Good catch! Updated in https://github.com/woocommerce/woocommerce/pull/38405/commits/1378d074669f4cd0f440c845ee2aa00c1671430c